commitizen/commands/bump.py

```diff
@@ -209,16 +213,22 @@ def __call__(self):  # noqa: C901
                 },
             )
             changelog_cmd()
-            c = cmd.run(f"git add {changelog_cmd.file_name}")
+            c = cmd.run(f"git add {changelog_cmd.file_name} {' '.join(version_files)}")
 
         self.config.set_key("version", str(new_version))
 
         if is_files_only:
             raise ExpectedExit()
 
         c = git.commit(message, args=self._get_commit_args())
+        if self.retry and c.return_code != 0 and self.changelog:
+            # Maybe pre-commit reformatted some files? Retry once
+            logger.debug("1st git.commit error: %s", c.err)
+            logger.info("1st commit attempt failed; retrying once")
+            cmd.run(f"git add {changelog_cmd.file_name} {' '.join(version_files)}")
+            c = git.commit(message, args=self._get_commit_args())
         if c.return_code != 0:
-            raise BumpCommitFailedError(f'git.commit error: "{c.err.strip()}"')
+            raise BumpCommitFailedError(f'2nd git.commit error: "{c.err.strip()}"')
```

Comment:
Could we add a test case that the 2nd failed?

Reply:
Test added.

Reply:
Thanks!